draft-ietf-rats-architecture.md

```diff
@@ -423,8 +423,10 @@ in {{trustmodel}} is also suitable for the local verifier.
 
 # Topological Models {#overview}
 
-There are multiple possible models for communication between an Attester,
-a Verifier, and a Relying Party.  This section includes some reference models,
+{{dataflow}} shows a basic model for communication between an Attester,
+a Verifier, and a Relying Party. The Attester sends its Evidence to the Verifier
+for appraisal, and the Verifier sends the Attestation Results to the Relying Party.
```

Comment:
I disagree with the use of "sends" with Verifier as the subject.   That implies how a particular conveyance protocol works, which is too restrictive and not the point of this section in my opinion.

Reply:
OK. I only want to clarify that a basic communication model will be the Attester communicates with the Verifier and the Verifier communicates with the Relying Party.
So how about to say "the Relying Party gets the Attestation Results from the Verifier"?